docs/src/main/sphinx/release/release-426.md

```diff
@@ -0,0 +1,39 @@
+# Release 426 (30 Aug 2023)
+
+## General
+
+* Fix event listener incorrectly reporting output columns for `UPDATE`
+  statements with subqueries. ({issue}`18815`)
+
+## Security
+
+* Fix authentication failure with OAuth 2.0 when authentication tokens are
+  larger than 4 KB. ({issue}`18836`)
+
+## Delta Lake connector
+
+* Add support for the `TRUNCATE TABLE` statement. ({issue}`18786`)
+* Add support for the `CASCADE` option in `DROP SCHEMA` statements. ({issue}`18333`)
+* Fix writing an incorrect transaction log for partitioned tables with an `id`
+  or `name` column mapping mode. ({issue}`18661`)
+
+## Hive connector
+
+* Add the `hive.metastore.thrift.batch-fetch.enabled` configuration property,
+  which can be set to `false` to disable batch metadata fetching from the Hive
+  metastore. ({issue}`18111`)
+* Fix `ANALYZE` failure when row count stats are missing. ({issue}`18798`)
+* Fix the `hive.target-max-file-size` configuration property being ignored
+  when writing to sorted tables. ({issue}`18653`)
+* Fix error when reading large SequenceFile, RCFile, or Avro files. ({issue}`18837`)
```

Comment:
The original "Fix reading ..." seems more clear. If we want to expand it, "Fix query failure when reading" is better.